frontend/src/app/modules/boards/board/board-filter/board-filter.component.ts

```diff
@@ -11,27 +11,18 @@ import {WorkPackageViewFiltersService} from "core-app/modules/work_packages/rout
 import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-filter-instance-resource";
 import {UrlParamsHelperService} from "core-components/wp-query/url-params-helper";
 import {StateService} from "@uirouter/core";
-import {DebouncedEventEmitter} from "core-components/angular/debounced-event-emitter";
-import {skip} from "rxjs/internal/operators";
-import {ApiV3Filter} from "core-components/api/api-v3/api-v3-filter-builder";
+import {debounceTime, skip, take} from "rxjs/internal/operators";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
-import {componentDestroyed} from "@w11k/ngx-componentdestroyed";
+import {Observable} from "rxjs";
+import {BoardFiltersService} from "core-app/modules/boards/board/board-filter/board-filters.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/9fa518c8247ead78913e82abf903114474bb7670/frontend/src/app/modules/boards/board/board-filter/board-filter.component.ts#L18"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>